kuka_ros2_control_sim_support/urdf/common_sim_kuka.xacro

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" ?>
+<robot name="kuka" xmlns:xacro="http://wiki.ros.org/xacro">
+
+  <xacro:arg name="prefix" default="" />
+  <xacro:arg name="use_mock_hardware" default="false" />
+  <xacro:arg name="sim_gazebo_classic" default="false" />
+  <xacro:arg name="sim_gazebo" default="false" />
+  <xacro:arg name="simulation_controllers" default="" />
+  <xacro:arg name="robot_description_package" default="kuka_kr3_support"/>
+  <xacro:arg name="robot_description_macro_file" default="kr3r540_macro.xacro"/>
+  <xacro:arg name="robot_name" default="kuka_kr3r540"/>
+
+  <xacro:property name="robot_description_package" value="$(arg robot_description_package)"/>
+  <xacro:property name="robot_description_macro_file" value="$(arg robot_description_macro_file)"/>
+  <xacro:property name="robot_name" value="$(arg robot_name)"/>
+
+   <!-- create link fixed to the "world" -->
+   <!-- using-external-joints-and-links approach -->
+  <link name="world" />
+  <joint name="$(arg prefix)base_joint" type="fixed">
+    <origin xyz="0 0 0" rpy="0 0 0" />          <!-- position robot in the world -->    
+    <parent link="world" />
+    <child link="$(arg prefix)base_link"/>
+  </joint>  
+
+  <xacro:include filename="$(find ${robot_description_package})/urdf/${robot_description_macro_file}"/>
+
+  <xacro:include filename="$(find kuka_ros2_control_sim_support)/urdf/common_sim_macro.ros2_control.xacro"/>
+
+
+  <xacro:kuka_common_sim_ros2_control_macro
+    name="$(arg robot_name)"
+    prefix="$(arg prefix)"
+    use_mock_hardware="$(arg use_mock_hardware)"
+    sim_gazebo_classic="$(arg sim_gazebo_classic)"
+    sim_gazebo="$(arg sim_gazebo)"
+    simulation_controllers="$(arg simulation_controllers)"
+    />
+    
+  <xacro:if value="${robot_description_package == 'kuka_kr3_support'}">
+    <xacro:kuka_kr3r540 prefix="$(arg prefix)"/>
```

Comment:
not perfect, but I suspect that nothing else worked...

Reply:
Let's keep it like this. Maybe we add multiple files at some point.

Reply:
> not perfect, but I suspect that nothing else worked...

yes tried ${}, as arguments call and as property call,